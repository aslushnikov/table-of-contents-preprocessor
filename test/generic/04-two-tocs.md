# TOC - 1

@@TOC@@

# TOC - 2

@@TOC@@
