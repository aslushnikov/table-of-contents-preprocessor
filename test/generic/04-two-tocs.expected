# TOC - 1

- [TOC - 1](#toc---1)
- [TOC - 2](#toc---2)

# TOC - 2

- [TOC - 1](#toc---1)
- [TOC - 2](#toc---2)
